packages/preferences/src/browser/views/preference-widget-bindings.ts

```diff
@@ -24,8 +24,11 @@ import { PreferencesSearchbarWidget } from './preference-searchbar-widget';
 import { PreferencesScopeTabBar } from './preference-scope-tabbar-widget';
 import { PreferencesDecorator } from '../preferences-decorator';
 import { PreferencesDecoratorService } from '../preferences-decorator-service';
+import { PreferenceTreeModel } from '../preference-tree-model';
 
 export function bindPreferencesWidgets(bind: interfaces.Bind): void {
+    let preferencesTreeContainer: Container;
```

Comment:
This issue can potentially be avoided by creating a subcontainer for PreferenceWidget and binding all preference sub-widgets/elements within that container

Reply:
Good idea 😃

Reply:
I've implemented the bindings in this way, and it seems everything is working and the bindings are cleaner.